src/Common/src/System/MathF.netstandard.cs

```diff
@@ -52,5 +52,11 @@ public static float Tan(float x)
         {
             return (float)Math.Tan(x);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static float Pow(float x, float y)
```

Comment:
nit: Keep the methods ordered alphabetically, it makes it easier to locate as the class grows in size.